Description: Add a directory entry to the info page.
 This patch seems to have always been part of the package but its
 origin is undocumented.
Author: Debian

--- time-1.7.orig/time.texi
+++ time-1.7/time.texi
@@ -11,6 +11,12 @@
 @finalout
 @end iftex
 
+@dircategory Individual utilities
+@direntry
+* time: (time).                     Run programs and summarize
+                                    system resource usage.
+@end direntry
+
 @ifinfo
 This file documents the the GNU @code{time} command for running programs
 and summarizing the system resources they use.
